tests/integration-tests/tests/integration/sqlite_authority_tests.rs

```diff
@@ -1392,3 +1392,28 @@ async fn test_refused_axfr() {
     // just update this if the count goes up in the authority
     assert!(result.unwrap_err().is_refused());
 }
+
+#[tokio::test]
+async fn test_axfr_deny_unsigned() {
```

Comment:
And postpone the introduction of this test for a separate commit after the positive case for `AllowSigned` is supported.